### DISCUSSION TITLE
What are the thoughts on moving the code to monorepo.

### DISCUSSION THREAD

**Discussion by rubiin** [rubiin/ultimate-nest#202] [Announcements]
Moving the code to monorepo so that I can integrate a frontend ui as well. Also might add a fastify version alongside express

**Comment by hi-zp** [comment]
It would be great to use NX!

**Reply by rubiin** [reply]
@hi-zp  you can the implementation at https://github.com/rubiin/nx

**Reply by hi-zp** [reply]
cool!
